muffnn/mlp/mlp_regressor.py

```diff
@@ -52,6 +52,13 @@ class MLPRegressor(MLPBaseEstimator, RegressorMixin):
         If int, the random number generator seed. If RandomState instance,
         the random number generator itself. If None, then `np.random` will be
         used.
+    solver: an instance of a subclass of `tf.train.Optimizer`, optional
+        The solver to use to minimize the loss. The default is
+        `tf.train.AdamOptimizer`.
+    **solver_kwargs: keyword arguments, optional
```

Comment:
i think this line should just be `**solver_kwargs` with no type.  i can't find many examples of this, but here's one: http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html